### DISCUSSION TITLE
Where is the launcher?

### DISCUSSION THREAD

**Discussion by Mrgoldilocks** [SKCraft/Launcher#502] [I need help!]
Hello yes, 

Am windows user, I am too dumb to find the .jar/exe file. hyelp me please.

Sincerely
ignorant fool

**Comment by hedgehog1029** [comment]
If you're trying to build the launcher, see here: https://github.com/SKCraft/Launcher/wiki/Initial-Steps#what-was-just-created

If you've arrived at this repository looking for a pre-built launcher: we don't have those here, this is the upstream repository designed for whiteboxing.

**Reply by Mrgoldilocks** [reply]
ooooh, ok. Thank you for clearing that up.
